src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp

```diff
@@ -84,8 +84,10 @@ void ToolbarAdapter::cleanToolItem(GtkToolItem* it) {
     if (data) {
         gtk_widget_set_sensitive(GTK_WIDGET(it), ToolitemDragDrop::isToolItemEnabled(data));
     }
+    GdkWindow* window = gtk_widget_get_window(GTK_WIDGET(it));
 
-    gdk_window_set_cursor(gtk_widget_get_window(GTK_WIDGET(it)), nullptr);
+    if (window)
+        gdk_window_set_cursor(window, nullptr);
```

Comment:
Just one thing, to match our coding conventions:
```suggestion
    if (window) {
        gdk_window_set_cursor(window, nullptr);
    }
```